workers_launch_template.tf

```diff
@@ -229,6 +229,14 @@ resource "aws_launch_template" "workers_launch_template" {
     )
   }
 
+  dynamic instance_market_options {
```

Comment:
Awesome! Finally seeing some great use of the new features in TF 0.12. Elegant 💅